test/graphql/types/Organization/updater.test.ts

```
@@ -0,0 +1,260 @@
+  // Define more specific type for operators
+  type Operators = {
+    eq: (field: TableFields[keyof TableFields], value: any) => SQL;
+  };
+
+type DeepPartial<T> = {
+	[P in keyof T]?: T[P] extends object ? DeepPartial<T[P]> : T[P];
+};
+const OrganizationResolver = {
+    updater: async (
+      parent: Organization, 
+      _args: {}, 
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)